experimental/runners/LinalgComprehensiveBufferizePass.cpp

```diff
@@ -1357,12 +1383,12 @@ static LogicalResult convertSubTensorOp(OpBuilder &b, SubTensorOp subTensor,
   auto dstTensorType = subTensor.result().getType().cast<RankedTensorType>();
 
   auto subviewMemRefType =
-      SubViewOp::inferRankReducedResultType(
+      memref::SubViewOp::inferRankReducedResultType(
           dstTensorType.getRank(), srcMemrefType, subTensor.getMixedOffsets(),
           subTensor.getMixedSizes(), subTensor.getMixedStrides())
           .cast<MemRefType>();
 
-  Value subView = b.create<SubViewOp>(
+  Value subView = b.create<memref::SubViewOp>(
       loc, subviewMemRefType, srcMemref, subTensor.getMixedOffsets(),
       subTensor.getMixedSizes(), subTensor.getMixedStrides());
```

Comment:
I know this is not related to this patch. Just wanna point out that you don't have to infer result type. There is a builder doing this for you if you don't pass the result type to it.